src/admin_tools/main.py

```diff
@@ -93,13 +94,13 @@ def _reindex(args):
         raise ValueError("--ref value should be in the format '1/2', '1/2/3', or '1'")
     reindexing_obj = len(id_pieces) >= 2
     if reindexing_obj:
-        ev = {'evtype': 'INDEX_NONEXISTENT', 'wsid': id_pieces[0], 'objid': id_pieces[1]}
+        ev = {'evtype': 'INDEX_NONEXISTENT', 'wsid': int(id_pieces[0]), 'objid': int(id_pieces[1])}
```

Comment:
Wrapping all this in `int()` doesn't really affect anything in the stack, but seems like good practice.